modules/efk-logging-deploying-storage-considerations.adoc

```diff
@@ -0,0 +1,108 @@
+// Module included in the following assemblies:
+//
+// * logging/efk-logging-deploy.adoc
+
+[id="efk-logging-deploy-storage-considerations-{context}"]
+= Storage considerations for cluster logging and {product-title}
+
+An Elasticsearch index is a collection of primary shards and its corresponding replica
+shards. This is how ES implements high availability internally, therefore there
+is little need to use hardware based mirroring RAID variants. RAID 0 can still
+be used to increase overall disk performance.
+
+//Following paragraph also in nodes/efk-logging-elasticsearch
+
+Elasticsearch is a memory-intensive application. Each Elasticsearch node needs 16G of memory for both memory requests and CPU limits 
```

Comment:
I would not say 'needs' but somethign along the lines of "Cluster Logging" defaults each Elasticsearch node with ..... We may not even want to mention a specific number or point them to a place in the code.  This way we don't need to redocument  if/when it changes.

Reply:
Chnaged to _The default cluster logging installation deploys 16G of memory for both memory requests and CPU limits._